boot/zephyr/Kconfig

```diff
@@ -303,6 +303,20 @@ endif
 
 endchoice
 
+config BOOT_SIGNATURE_USING_KMU
+	bool "Use KMU stored keys for signature verification"
+	select PSA_WANT_ALG_GCM
+	select PSA_WANT_KEY_TYPE_AES
+	select PSA_WANT_AES_KEY_SIZE_256
+	select PSA_WANT_ALG_SP800_108_COUNTER_CMAC
+	select PSA_WANT_ALG_CMAC
+	select PSA_WANT_ALG_ECB_NO_PADDING
+	help
```

Comment:
this should have a depends on so it can only be enabled on hardware that supports it

Reply:
`CRACEN_LIB_KMU` is a candidate for a depends. It is default enabled for nRF54L15 devices and it is the signal to allow for 

This Kconfig is available [here](https://github.com/nrfconnect/sdk-nrf/blob/7064aee2128799895ed5357f886a70ece2ec9f28/subsys/nrf_security/src/drivers/cracen/Kconfig#L26)